### ISSUE TITLE
[emmet] http-equiv="X-UA-Compatible" in 2023

### ISSUE THREAD

**Issue by wesbos** [microsoft/vscode#165933]
The emmet expansions `!` and `html:5` both produce this expansion:

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
</head>
<body>
  
</body>
</html>
```

The `<meta http-equiv="X-UA-Compatible" content="IE=edge">` meta tag is unnecessary and should be removed. It was [last used in IE10](https://stackoverflow.com/questions/26346917/why-use-x-ua-compatible-ie-edge-anymore) to turn on compatibility mode and now as even IE11 is dead, we can safely remove it.

It was initially removed in 2019: https://github.com/microsoft/vscode/issues/80923
Then added back in: https://github.com/microsoft/vscode/issues/108119
Then suggested to remove it again in 2021, which Im pretty sure incorrectly says it was needed for ie11: https://github.com/microsoft/vscode/issues/128374

Now that all IE is dead, let's get rid of it!

**Comment by jimmywarting** [comment]
Also think it's time!

**Comment by wesbos** [comment]
The charset is needed. It's not the default over http, so many characters are broken without - including all emojis

**Comment by wesbos** [comment]
Looks like this change has been merged into Emmet! Hopefully can make it into a VS Code release soon

**Comment by wesbos** [comment]
Okay folks, Emmet just published a new version with this included.

Now VS Code requires Emmet Module: https://github.com/microsoft/vscode/tree/7b83bc62ad9ce1d141512f8ec67dda7615765f88/extensions/emmet

which in turn requires Emmet Helper: https://github.com/microsoft/vscode-emmet-helper/blob/main/package.json#L26

I think according to their versions, it should just make it into the next version of VS Code? Or do those upstream packages need to update as well?

**Comment by rzhao271** [comment]
Blocked on https://github.com/emmetio/math-expression/issues/11
